### ISSUE TITLE
Support variable fonts

### ISSUE THREAD

**Issue by MaxValue** [CollaboraOnline/online#10569]
### Is your feature request related to a problem?

Yes, variable fonts seem the modern way to include typefaces with multiple variants (e.g. italic/bold/thin), because it only requires you to include 1 file instead of multiple files (one per variant).

### Describe the solution you'd like

When a custom (variable) font is installed, the system recognizes it as such and renders all the variants in the font selection dropdown.

### Describe alternatives you've considered

Installing fonts the classical way: install 1 file per variant.

### Additional context

At the moment, the system just displays the first variant in the file. In my case I tried to include Montserrat and it only displayed the Thin variant, because this is the first in the list of variant options of this font.

**Comment by timur-g** [comment]
That seems like Core issue https://bugs.documentfoundation.org/show_bug.cgi?id=155853.

**Comment by sebastiaanveld** [comment]
Looks fixed upstream in LO to be released in 26.8
https://bugs.documentfoundation.org/show_bug.cgi?id=155853

Is this fix in CO 26.04?

**Comment by timur-g** [comment]
No, since code base was split everything interested should be manually ported to CO, even from 26.2, moreover from 26.8. Let's see.
